From 584fe5187b36d633fc24f0cb3e7a89045d6f9ae3 Mon Sep 17 00:00:00 2001
From: Moses <loschnigsolutions@gmail.com>
Date: Sun, 13 Sep 2020 16:30:28 +0530
Subject: [PATCH] test


diff --git a/while/wh7.c b/while/wh7.c
index 9a3c88e..1f0cfc5 100644
--- a/while/wh7.c
+++ b/while/wh7.c
@@ -1,24 +1,28 @@
 
-
-
 #include<stdio.h>
+
 void main()
 {
-	int n,i,row=1;
+	int n,i,row;
 	char ch ='*';
+
 	printf("enter value for row:\n");
 	scanf("%d",&n);
-	printf("*\n");
-	while(row<=n)
+
+	row = n;
+
+	printf("\n");
+
+	while(row >= 0)
+	{
+		i=1;
+		while(i<=row)
 		{
-			i=1;
-			while(i>=row)
-			{
-				i--;
-				printf("%c",ch);
-			}
-		printf("\n");
-		row++;
+			i++;
+			printf("%c",ch);
 		}
+		printf("\n");
+		row--;
+	}
 }
